### PR TITLE
css->scss : banner + opensource

### PR DESCRIPTION
**Pull request by ashuvssut**
adds to #679 

I have used dart-sass instead of node-sass

dart-sass will have all the major updates and node-sass will not get the major updates

for example: we when importing variables, dart-sass gives `@use` which offers more features that `@import`

node-sass doesnt support `@use`

`@import` is going to get deprecated on oct2021.

so i used dart-sass and thus removed node-sass

to install dart-sass : `npm i sass`

--------------------------------------
_- I have participated in GSSOC and CrossWoC_

### CODE REVIEW

File: src/pages/Home/sections/Banner/Banner.module.scss [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  .wave4 {
+    animation: animate2 5s linear infinite;
+    z-index: 997;
+    opacity: 0.7;
+    animation-delay: -5s;
+    bottom: 20px;
+  }
+}
+
+@keyframes animate {
```
Comment (anushbhatia) [inline src/pages/Home/sections/Banner/Banner.module.scss:139]: why two similar animations have same name

File: src/pages/Home/sections/OpenSource/OpenSource.module.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
+.OpenSource {
+  text-align: center;
+  margin-top: 7rem;
+
+  .title {
+    font-family: "Poppins", sans-serif;
+    color: #4e89ae;
+    font-size: 3rem;
+    font-weight: bold;
+    text-shadow: 3px 4px 2px #0000002e;
```
Comment (anushbhatia) [inline src/pages/Home/sections/OpenSource/OpenSource.module.scss:10]: colour code is not hexadecimal

File: src/scss/_colors.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+$white: #fff;
```
Comment (anushbhatia) [inline src/scss/_colors.scss:1]: Define all the colours on a map 
https://sass-lang.com/documentation/variables
refer to the last section of this.
why is only a single colour imported here?

**Review by anushbhatia — CHANGES_REQUESTED** [review]

File: src/pages/Home/sections/OpenSource/OpenSource.module.scss (hunk 2 above)
Comment (ashuvssut) [reply src/pages/Home/sections/OpenSource/OpenSource.module.scss:10]: `#000000` represents black `2e` represents the opacity

Comment (ashuvssut) [reply src/pages/Home/sections/OpenSource/OpenSource.module.scss:10]: `#0000002e` is a valid color

File: src/pages/Home/sections/Banner/Banner.module.scss (hunk 1 above)
Comment (ashuvssut) [reply src/pages/Home/sections/Banner/Banner.module.scss:139]: `animate` and `animate2 are different. see carefully. `background-position-x` is different

**Pull request merged**